itwin-platform-access/imodels-access-backend/package.json

```diff
@@ -50,8 +50,8 @@
     "typescript": "~4.4.0"
   },
   "peerDependencies": {
-    "@itwin/core-backend": "^3.3.0",
-    "@itwin/core-bentley": "^3.3.0",
-    "@itwin/core-common": "^3.3.0"
+    "@itwin/core-backend": "^3.6.0-dev.21",
+    "@itwin/core-bentley": "^3.6.0-dev.21",
+    "@itwin/core-common": "^3.6.0-dev.21"
```

Comment:
Maybe we should wait until 3.6.0 is officially released since the dev packages can be weird as peerDependencies, but I wanted to get some feedback sooner rather than later.